atcoder-problems-frontend/src/pages/Internal/UserProblemListPage.tsx

```diff
@@ -86,15 +81,19 @@ export const UserProblemListPage = connect<unknown, Props>(() => ({
                 internalListId={internal_list_id}
                 internalListName={internal_list_name}
                 listItemCount={items.length}
-                deleteList={props.deleteList}
+                deleteList={async (internalListId) => {
+                  await deleteList(internalListId).then(() =>
+                    myListFetch.mutate()
+                  );
```

Comment:
async function になっているので以下のようにしても良さそうですね
```suggestion
                  await deleteList(internalListId);
                  await myListFetch.mutate();
```